modules/oncodrivefml/1.0/etc/oncodrivefml.conf

```diff
@@ -0,0 +1,33 @@
+[genome]
+build = 'hg19'
+
+[signature]
+method = 'complement'
+
+[score]
+# score file path must be the directory containing scoe files
+file = 
+format = 'pack'
+chr = 0
+chr_prefix = ""
```

Comment:
same comment as above regarding possible need of adjustment for hg38?

Reply:
I added more information to this section as well